controllers/products.js

```
@@ -17,15 +17,20 @@ ProductRouter.post(
       return res.status(400).json({ errors: errors.array() });
     }
     const { product_name, unit_price, quantity, user_id } = req.body;
+    const productRevenue = unit_price * quantity
 
     const existingUser = await User.findById(user_id);
 
+    if (!existingUser) {
```

this would blow the app up, i think you should just throw an error response. like this;
```javascript
if(!existingUser) return res.json({ error: "User not found." })
```

although this edge case would almost never be needed as the authMiddleware doesnt even let the response get to the app.